packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawer.tsx

```diff
@@ -105,8 +105,8 @@ export const NavigationDrawer = ({
           !isMobile && <NavigationDrawerBackButton title={title} />
         ) : (
           <NavigationDrawerHeader
-            name={title}
-            logo={logo}
+            placeholderName={title}
```

Comment:
We had discussed orally that the name change was for the variables (the big base64) not the component variables (you said yourself it didn't make sense :))

Reply:
I think it's not needed as we separated it to dropdown component altogether, let me know if you think it still makes sense to name the variable differently :)